src/main/java/io/supertokens/multitenancy/MultitenancyHelper.java

```diff
@@ -75,7 +75,8 @@ public static void init(Main main) throws StorageQueryException, IOException {
                         new TenantConfig(
                                 new TenantIdentifier(null, null, null),
                                 new EmailPasswordConfig(true), new ThirdPartyConfig(true, null),
```

Comment:
missing in this PR is the implementation of adding factors for a user.

Reply:
not required